src/components/EmailValidationForm.tsx

```diff
@@ -54,7 +54,7 @@ export class EmailValidationForm extends React.Component<EmailValidationFormProp
 
     return (
         <Form
-          className="validation"
+          className="validation border"
```

Comment:
It seems that this `border` class appears all the time. It makes more sense to have this be the default and make the dashed border an optional class.

Reply:
The dashed border isn't a real class; I just used it for the Storybook examples, because it's otherwise hard to visually differentiate between the different forms when they're all in a list like that.  I initially had the default be border (and had a `no-border` class), but I changed it because I noticed that, in circ-web, the overwhelming majority of the forms needed to be `no-border`, so I figured that the much much more common use case should be the default...

Reply:
Sounds fine.